.travis.yml

```diff
@@ -1,4 +1,6 @@
 language: node_js
 node_js:
     - "0.10"
-
+    - "0.12"
```

Comment:
I believe it's important to make sure the development environment is compatible to latest version of node.js and io.js